packages/x-data-grid-premium/src/components/GridColumnMenuAggregationItem.tsx

```diff
@@ -64,34 +68,68 @@ function GridColumnMenuAggregationItem(props: GridColumnMenuItemProps) {
 
   const label = apiRef.current.getLocaleText('aggregationMenuItemHeader');
 
+  const handleMenuItemKeyDown = React.useCallback((event: React.KeyboardEvent) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      inputRef.current.focus();
+    }
+  }, []);
+
+  const handleSelectKeyDown = React.useCallback((event: React.KeyboardEvent) => {
+    if (event.key === 'ArrowDown' || event.key === 'ArrowUp') {
```

Comment:
To avoid this:

https://github.com/user-attachments/assets/8be49761-596c-4bc7-8635-c13cecc36103

```suggestion
    if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === ' ') {
```